spec/factories/mendeley_papers.rb

```diff
@@ -0,0 +1,10 @@
+FactoryGirl.define do
+  factory :mendeley_paper do
+    title "Musterstudie"
+    uuid { UUIDTools::UUID.random_create.to_s }
+    first_author "Max Mustermann"
+    mendeley_url "http://example.com"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.